libs/sync/src/lib/sync-details-page/sync-modal/sync-modal.component.html

```diff
@@ -0,0 +1,20 @@
+<pa-modal-dialog>
+  <pa-modal-title>
+    <h2>{{ 'sync.details.modal.title' | translate }}</h2>
```

Comment:
you don't need this `h2`: `pa-modal-title` is projecting its content directly in a properly styled tag.